cognite_toolkit/_cdf_tk/client/api/fixed_transformations.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+import gzip
+from typing import Any
+
+from cognite.client._api.transformations import TransformationsAPI
+from cognite.client.config import global_config
+from cognite.client.data_classes import TransformationPreviewResult
+from cognite.client.utils import _json
+from requests import Response
+
+
+class FixedTransformationsAPI(TransformationsAPI):
+    """This overrides the preview method in the transformations API to allow for longer timeouts and retries."""
+
+    _default_timeout_run_query = 240  # seconds, this is the maximum timeout for running queries in CDF
+
+    def preview(
```

Comment:
Note that this is well tested through the `tests/test_integration/test_utils/test_cdf.py` tests.